appengine/examples.py

```diff
@@ -104,3 +99,74 @@ def get_example(name):
 examples = { 'datalog' : datalog_examples,
              'myrial' : myria_examples,
              'sql' : sql_examples }
+
+
+
+load_twitterk_data = '''T1 = load("s3://uwdb/sampleData/TwitterK.csv",
```

Comment:
None of these S3 URLs will work on installs without AWS credentials present (which will always be the case on EC2 unless you specify `--role` to `myria-cluster` or somehow set credentials yourself). A [patch](https://issues.apache.org/jira/browse/HADOOP-13237) just landed in Hadoop 2.8 to fix this, but we'd have to either wait for the 2.8 release or backport the patch to our custom maven repo for Hadoop 2.7.2. If it's important enough, I'd be happy to make the backport.